tables.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE `release`
+(
+    `id` INTEGER NOT NULL AUTO_INCREMENT,
+    `name` VARCHAR(255) NOT NULL,
+    `info` VARCHAR(255) NOT NULL,
+    PRIMARY KEY (`id`)
+) ENGINE=InnoDB;
```

Comment:
I think @alex1987 said sometime earlier that one of the utf8mb4 variant is the one we should use.